models/sipnet/R/write.restart.SIPNET.R

```diff
@@ -18,85 +18,82 @@
 ##' @return NONE
 ##' @export
 ##' 
-write.restart.SIPNET<- function(out.dir, runid, time, settings, analysis.vec,
-                                RENAME = TRUE, variables,
-                                sample_parameters = FALSE,
-                                trait.values = NA, met){
+write.restart.SIPNET<- function(outdir, runid, start.time, stop.time,
+                                settings, new.state,
+                                RENAME = TRUE,
+                                new.params = FALSE,
+                                inputs){
```

Comment:
Inconsistent variable names being passed between write.restart.SIPNET and write.restart.LINKAGES